agent/agent.go

```
@@ -447,6 +510,23 @@ func stopServiceInputs(inputs []*models.RunningInput) {
 	}
 }
 
+func (a *Agent) StopInputPlugin(input string) {
+
+	// will write "STOP" to the channel as a case for gatherLoop
+	status_channel, ok := a.inputStatus[input]
+	if !ok {
+		fmt.Errorf("E! [agent] You are trying to stop an input that is not running: %s", input)
```

Should probably return in this case?

Maybe return an error?